MAVProxy/modules/mavproxy_messagerate.py

```diff
@@ -18,16 +18,18 @@ def __init__(self, mpstate):
         self.counts = {}
         self.buckets = []
         self.max_buckets = 5
+        
+        self.sysid = (mpstate.settings.target_system, mpstate.settings.target_component)
```

Comment:
Unused?

Reply:
Unfortunate leftover from attempting to get param id from the param module.  Needs to be removed.

Reply:
Lines deleted from messagerate branch